x-pack/test/api_integration/apis/telemetry/telemetry.js

```diff
@@ -5,48 +5,108 @@
 import basicClusterFixture from './fixtures/basiccluster';
 
+/**
+ * Update the .monitoring-* documents loaded via the archiver to the recent `timestamp`
+ * @param esSupertest The client to send requests to ES
+ * @param fromTimestamp The lower timestamp limit to query the documents from
+ * @param toTimestamp The upper timestamp limit to query the documents from
+ * @param timestamp The new timestamp to be set
+ */
+function updateMonitoringDates(esSupertest, fromTimestamp, toTimestamp, timestamp) {
+  return Promise.all([
+    esSupertest
+      .post('/.monitoring-es-*/_update_by_query?refresh=true')
+      .send({
+        query: {
+          range: {
+            timestamp: {
+              format: 'epoch_millis',
+              gte: moment(fromTimestamp).valueOf(),
+              lte: moment(toTimestamp).valueOf(),
+            },
+          },
+        },
+        script: {
+          source: `ctx._source.timestamp='${timestamp}'`,
+          lang: 'painless',
+        },
+      })
+      .expect(200),
+    esSupertest
+      .post('/.monitoring-kibana-*/_update_by_query?refresh=true')
+      .send({
+        query: {
+          range: {
+            timestamp: {
+              format: 'epoch_millis',
+              gte: moment(fromTimestamp).valueOf(),
+              lte: moment(toTimestamp).valueOf(),
+            },
+          },
+        },
+        script: {
+          source: `ctx._source.timestamp='${timestamp}'`,
+          lang: 'painless',
+        },
+      })
+      .expect(200),
+  ]);
+}
+
 export default function ({ getService }) {
   const supertest = getService('supertest');
   const esArchiver = getService('esArchiver');
+  const esSupertest = getService('esSupertest');
 
   describe('/api/telemetry/v2/clusters/_stats', () => {
-    it('should load multiple trial-license clusters', async () => {
+    const timestamp = new Date().toISOString();
+    describe('monitoring/multicluster', () => {
       const archive = 'monitoring/multicluster';
-      const timestamp = '2017-08-16T00:00:00Z';
-
-      await esArchiver.load(archive);
-
-      const { body } = await supertest
-        .post('/api/telemetry/v2/clusters/_stats')
-        .set('kbn-xsrf', 'xxx')
-        .send({ timestamp, unencrypted: true })
-        .expect(200);
-
-      expect(body).length(3);
-      expect(body).to.eql(multiClusterFixture);
+      const fromTimestamp = '2017-08-15T21:00:00.000Z';
+      const toTimestamp = '2017-08-16T00:00:00.000Z';
+      before(async () => {
+        await esArchiver.load(archive);
+        await updateMonitoringDates(esSupertest, fromTimestamp, toTimestamp, timestamp);
+      });
+      after(() => esArchiver.unload(archive));
+      it('should load multiple trial-license clusters', async () => {
+        const { body } = await supertest
+          .post('/api/telemetry/v2/clusters/_stats')
+          .set('kbn-xsrf', 'xxx')
+          .send({ timestamp, unencrypted: true })
```

Comment:
The `timestamp` parameter was only used for the `monitoring` strategy to retrieve the right time-frame of telemetry data. Now it's useless and confusing. I'll create a follow-up PR to remove it.